txscript/script.go

```diff
@@ -69,6 +69,87 @@ func IsPayToScriptHash(script []byte) bool {
 	return isScriptHash(pops)
 }
 
+// isWitnessScriptHash returns true if the passed script is a
+// pay-to-witness-script-hash transaction, false otherwise.
+func isWitnessScriptHash(pops []parsedOpcode) bool {
```

Comment:
Please move this and all of the others that represent standard scripts into standard.go since this is not consensus critical code.